src/api/goods.ts

```diff
@@ -10,10 +10,12 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => [...goods].sort(
```

Comment:
Consider using a more descriptive name for the parameter `goods` in the function `get5First`. For example, `allGoods`. 🤔